tezos/encoding-derive/src/lib.rs

```diff
@@ -0,0 +1,662 @@
+extern crate proc_macro;
```

Comment:
@akoptelov 
please add here, `#![forbid(unsafe_code)]` and the licence stuff at the begining

Reply:
This is proc-macro module, so it does not contribute any code directly. Nevertheless, the attribute is added.

Reply:
aha, ok, sorry, I did not know